client/rest-client/src/main/java/org/infinispan/client/rest/RestSecurityClient.java

```diff
@@ -15,4 +15,8 @@
    CompletionStage<RestResponse> deny(String principal, List<String> roles);
 
    CompletionStage<RestResponse> flushCache();
```

Comment:
aren't we missing a CLI operation to flush the ACL cache?

Reply:
The cache is flushed automatically when there are changes. There is a jira for adding stats and ops https://issues.redhat.com/browse/ISPN-13743